source/playbooks/work-with-playbooks.rst

```diff
@@ -71,18 +71,29 @@ Make checklists
 
 5. Choose **Save**.
 
-Runs and channel behavior
--------------------------
+Create fewer channels
+---------------------
 
-When setting up your playbook:
+From Mattermost v7.7, you can choose to start each run in a new channel or re-use the same channel.
```

Comment:
```suggestion
From Mattermost v7.7, you can choose to start each run in a new channel or re-use an already existing channel.
```

not sure about the exact wording, but the concept of "same channel" seems a bit awkward because when I start reading the sentence, in my mind I'm not in the context of any channel. 

Additionally, the existing-channel / new-channel are the exact words that we use in the product. Not sure if there are the best, but I'd like to use similar expressions both product and docs.